testutil/keeper/observer.go

```diff
@@ -3,6 +3,8 @@ package keeper
 import (
 	"testing"
 
+	slashingkeeper "github.com/cosmos/cosmos-sdk/x/slashing/keeper"
+
```

Comment:
```suggestion
```
We should run `goimport` then

Reply:
This has been sorted by goimports already